Aplikacja/app/build.gradle

```diff
@@ -2,6 +2,22 @@ apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-android-extensions'
 
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+
+    dependencies {
+        classpath 'com.stanfy.spoon:spoon-gradle-plugin:1.2.0'

```

Comment:
move classpath to main build.gradle and remove whole buildscript from this file.